cage_moves:
  # - arm1_2_3_4/step1  
  # - arm1_2_3_4/step2  
  # - arm1_2_3_4/step3  
  # - arm1_2_3_4/step4  
  # - arm1_2_3_4/step5  
  - arm1_2_3_4/step6 # jacket approach
  # - arm1_2_3_4/step7 # jacket grab
  # - arm1_2_3_4/step8 # jacket lift
arm1_2_3_4: 
  step1: [-0.87,-0.87, 0.87,0,0,0,
           2.81, 0.87,-0.78,0,0,0,
          -2.81, 0.87,-0.78,0,0,0,
           0.87,-0.87, 0.87,0,0,0]
  step2: [-1.00,0.69,-0.69,0,0,0,
           2.35,1.22,-1.15,0,0,0,
          -2.35,1.22,-1.15,0,0,0,
           1.00,0.69,-0.69,0,0,0]
  step3: [-0.90,0.69,-0.69,0,0,0,
           2.25,1.22,-1.15,0,0,0,
          -2.25,1.22,-1.15,0,0,0,
           0.90,0.69,-0.69,0,0,0]
  step4: [-1.00,0.69,-0.69,0,0,0,
           2.45,1.22,-1.15,0,0,0,
          -2.45,1.22,-1.15,0,0,0,
           1.00,0.69,-0.69,0,0,0]
  step5: [-1.00,0.69,-0.69,0,0.2,0,
           2.35,1.22,-1.15,0,0.2,0,
          -2.35,1.22,-1.15,0,0.2,0,
           1.00,0.69,-0.69,0,0.2,0]
  step6: [0, -1.5707, 1.5, 0, 0.4, 0,
          0, -1.5707, 1.5, 0, 0.4, 0,
          -1.99, -0.57, 0.31, -0.08, 0.38, -1.55,
           2.09, -1.06, 0.91, 1.24, -0.02, 0.19]
  step7: [0, -1.5707, 1.5, 0, 0.4, 0,
          0, -1.5707, 1.5, 0, 0.4, 0,
          -2.07, -0.36, 0.24, -0.08, 0.19, -1.55,
           2.06, -0.52, 0.56, 1.24, -0.02, 0.19]
  step8: [0, -1.5707, 1.5, 0, 0.4, 0,
          0, -1.5707, 1.5, 0, 0.4, 0,
          -1.22, 0.31, 0.52, -0.08, -0.52, -1.64,
           1.27, -0.17, 0.80, 1.24, -0.37, 0.19]
  
  
  